checker-qual/src/main/java/org/checkerframework/checker/sqlquotes/qual/SqlEvenQuotes.java

```diff
@@ -0,0 +1,22 @@
+package org.checkerframework.checker.sqlquotes.qual;
+
+import java.lang.annotation.*;
+import org.checkerframework.framework.qual.QualifierForLiterals;
+import org.checkerframework.framework.qual.SubtypeOf;
+
+/**
+ * SqlEvenQuotes: Used to denote a String that comprises part of a SQL query and contains either
```

Comment:
The string isn't necessarily part of a SQL query.  It might be used that way or it might not, but the annotation is agnostic to such use.